src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts

```diff
@@ -259,4 +264,37 @@ export class GroupProgressGridComponent implements OnChanges, OnDestroy {
       });
   }
 
+  getCSVDownloadTypeByFilter(): 'group' | 'team' | 'user' {
+    switch (this.currentFilter) {
+      case 'Groups':
+        return 'group';
+      case 'Users':
+        return 'user';
+      case 'Teams':
+        return 'team';
+    }
+  }
+
+  onCSVExport(): void {
+    if (!this.group || !this.itemData) {
+      return;
+    }
```

Comment:
We should throw an error rather than returning

Reply:
Fixed